src/index.d.ts

```diff
@@ -15,8 +18,8 @@ export interface IOptions {
 
 // props specific to the React wrapper
 export interface SignatureCanvasProps extends IOptions {
-    canvasProps: any;
-    clearOnResize: boolean;
+    canvasProps?: any;
```

Comment:
shouldn't this be more specific? It takes an `object` with the same props as React's `canvas`. When I made it just `object` it complained that `width` and `height` weren't defined when used later.

See my `types` commit that makes this more specific. Not sure if `<object>` is the right type for the generic